src/assets/bogumil_love _m.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="418" height="394" viewBox="0 0 418 394" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I donf think that you need to store this image 3 times. Just transform it by CSS ;)

Reply:
@aniaorwatk up